pkg/envoy/xdsutil_test.go

```diff
@@ -23,6 +25,46 @@ func TestGetLocalClusterNameForService(t *testing.T) {
 	assert.Equal(actual, "default/bookbuyer-local")
 }
 
+func TestGetFileAccessLog(t *testing.T) {
+	assert := tassert.New(t)
+
+	expAccessLogger := &xds_accesslog.FileAccessLog{
```

Comment:
I understand this function is missing a unit test coverage, but testing a struct by constructing it the same way as the function under test isn't useful, since you likely copy-pasted this code from the function under test.

Reply:
@shashankram Would it be better to just check that it's not nil?

Reply:
I think this is okay, just that it doesn't do any testing of the actual function, so testing against nil would pretty much have the same effect as copy-pasting the actual function code into the test body. At the very least, I would test individual fields against hardcoded values instead of generating it the exact same way the function under test generates it.

Reply:
@shashankram, when you say hardcoded values do you mean like replacing pbStringValue(`%START_TIME%`) to 
&structpb.Value{
Kind: &structpb.Value_StringValue{
StringValue: `%START_TIME%`,
}

Reply:
Not for this example, because that doesn't give any additional benefit either. In other cases where struct fields are standard types, then yes. I think overall this change is okay, just that I personally don't see much value in testing `foo(x) == foo(x)` scenarios.

Reply:
@shashankram ah ok I understand what you mean